src/modules/users-login-routes/index.js

```diff
@@ -34,32 +38,32 @@ export default class UsersLoginRoutes extends HasUserModel {
     var email = req.param('email')
     return this.models.User.findOne({email}).then((user) => {
       if(!user) throw new Error('No user matching that email was found.')
-      var link = "http://"+this.app.config.baseUrl+"/login-link?token="+user.resetPasswordToken
+      var link = "http://"+this.app.config.this.baseUrl+this.baseUrl+"login-link?token="+user.resetPasswordToken
```

Comment:
search&replace typo - can you change these to `application.config.baseUrl+this.baseUrl`